### ISSUE TITLE
Package includes pip-installed packages: six + webencodings

### ISSUE THREAD

**Issue by tjd2002** [conda-forge/bleach-feedstock#17]
Issue:
Bleach 3.0.0-3.0.2 conda packages contain extra, pip-installed copies of `six` and `webencodings`.

This is apparently due to this commit from @ocefpaf : https://github.com/conda-forge/bleach-feedstock/commit/40ff6330800c7a90cd7ab861938138aa3d5c280e, which removed the `--no-deps` and `--ignore-installed` options to pip install.

Perhaps this was done in anticipation of conda-build providing these flags automatically as env vars, but if so it seems to have been premature.

You can confirm that this is the case by inspecting the tarball at https://anaconda.org/conda-forge/bleach/files, or by installing it and looking at the installed package.

```
conda create -c conda-forge -n bleach-issue bleach

tree -d $_CONDA_ROOT/pkgs/bleach-3.0.2-py_0/
/Users/XXX/conda/pkgs/bleach-3.0.2-py_0/
├── info
│   ├── recipe
│   └── test
└── site-packages
    ├── bleach
    │   └── _vendor
    │       ├── html5lib
    │       │   ├── _trie
    │       │   ├── filters
    │       │   ├── treeadapters
    │       │   ├── treebuilders
    │       │   └── treewalkers
    │       └── html5lib-1.0.1.dist-info
    ├── bleach-3.0.2.dist-info
    ├── six-1.11.0.dist-info
    ├── webencodings
    └── webencodings-0.5.1.dist-info

17 directories

``` 

<br/>
Environment (<code>conda list</code>):
<details>

```
$ conda list
# packages in environment at /Users/XXX/conda/envs/bleach-issue:
#
# Name                    Version                   Build  Channel
bleach                    3.0.2                      py_0    conda-forge
ca-certificates           2018.11.29           ha4d7672_0    conda-forge
certifi                   2018.11.29            py37_1000    conda-forge
libffi                    3.2.1                hfc679d8_5    conda-forge
ncurses                   6.1                  hfc679d8_2    conda-forge
openssl                   1.0.2p               h470a237_1    conda-forge
pip                       18.1                  py37_1000    conda-forge
python                    3.7.1                h46c1a51_0    conda-forge
readline                  7.0                  haf1bffa_1    conda-forge
setuptools                40.6.3                   py37_0    conda-forge
six                       1.12.0                py37_1000    conda-forge
sqlite                    3.26.0               hb1c47c0_0    conda-forge
tk                        8.6.9                ha92aebf_0    conda-forge
webencodings              0.5.1                      py_1    conda-forge
wheel                     0.32.3                   py37_0    conda-forge
xz                        5.2.4                h470a237_1    conda-forge
zlib                      1.2.11               h470a237_3    conda-forge
```
</details>

<br/>
Details about  <code>conda</code> and system ( <code>conda info</code> ):
<details>

```
$ conda info

     active environment : None
            shell level : 0
       user config file : /Users/XXX/.condarc
 populated config files : /Users/XXX/.condarc
          conda version : 4.5.11
    conda-build version : 3.16.3
         python version : 3.7.0.final.0
       base environment : /Users/XXX/conda  (writable)
           channel URLs : https://conda.anaconda.org/conda-forge/osx-64
                          https://conda.anaconda.org/conda-forge/noarch
                          https://repo.anaconda.com/pkgs/main/osx-64
                          https://repo.anaconda.com/pkgs/main/noarch
                          https://repo.anaconda.com/pkgs/free/osx-64
                          https://repo.anaconda.com/pkgs/free/noarch
                          https://repo.anaconda.com/pkgs/r/osx-64
                          https://repo.anaconda.com/pkgs/r/noarch
                          https://repo.anaconda.com/pkgs/pro/osx-64
                          https://repo.anaconda.com/pkgs/pro/noarch
          package cache : /Users/XXX/conda/pkgs
                          /Users/XXX/.conda/pkgs
       envs directories : /Users/XXX/conda/envs
                          /Users/XXX/.conda/envs
               platform : osx-64
             user-agent : conda/4.5.11 requests/2.19.1 CPython/3.7.0 Darwin/18.0.0 OSX/10.14
                UID:GID : 501:20
             netrc file : None
           offline mode : False
```
</details>

**Comment by tjd2002** [comment]
This seems to be related to:
https://github.com/conda-forge/conda-forge.github.io/pull/659
https://github.com/conda-forge/conda-forge.github.io/pull/653

These PRs suggest that the change was deliberate, and yet... this package still contains extra pip-installed packages. Maybe it just needs to be re-rendered? @jakirkham ?

**Comment by tjd2002** [comment]
OK, it's an interaction between conda-forge recently recommending removal of the --no-deps flag, and a short-lived bug in conda-build (introduced in https://github.com/isuruf/conda-build/commit/12975ecf237335908e05061b4ef50134add053c0, fixed in 3.17 with 
https://github.com/conda/conda-build/pull/3271/commits/43874ea9da6df495bffe7132aa704ccbf0957eac). So, just re-building the package with 3.17 should fix it.

Maybe I'll issue a PR that just bumps the build #?

**Comment by tjd2002** [comment]
In conda-build, the buggy flag was present starting at 3.12.1:
```
$ git tag --contains 12975ecf237335908e05061b4ef50134add053c0
3.12.1
3.13.0
3.14.0
3.14.1
3.14.2
3.14.3
3.14.4
3.15.0
3.15.1
3.16.0
3.16.1
3.16.2
3.16.3
3.17.0
3.17.1
3.17.2
3.17.3
3.17.4
```

The fix was introduced in 3.17.0:
```
$ git tag --contains 43874ea9da6df495bffe7132aa704ccbf0957eac
3.17.0
3.17.1
3.17.2
3.17.3
3.17.4
```
